woocommerce/woocommerce
# [Experimental] Create Quantity Selector block

### Changes proposed in this Pull Request:

This PR create the Quantity Selector experimental block.

Closes #53009.

Note: this PR doesn't include tests yet, but that's because the current behavior won't be final, so I think it's better to wait until all blocks work together.

### How to test the changes in this Pull Request:

1. Install and activate the `WooCommerce Beta Tester` plugin
2. Go to `WooCommerce Admin Test Helper` -> `Features` and enable `blockified-add-to-cart`.
3. Create a new product
4. Go to Pages > Add New Page
5. Add a `Single product` block and select the product you just created.
6. Go to the block inserter and add a `Quantity Selector` block.
7. Verify it looks correctly both in the editor and the frontend.
8. Switch between the _Input_ and _Stepper_ styles to verify it looks well in both cases.
9. To test the _Stepper_ style, add the _Add to Cart with Options (Experimental)_ block in the same template. Then, in the frontend, click the plus and minus icons to verify they update the input value.
